code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -592,7 +606,7 @@ Then we space some of our heat, and think about if we should stop conducting.
 	//Conduct with air on my tile if I have it
 	if(!blocks_air)
 		temperature = air.temperature_share(null, thermal_conductivity, temperature, heat_capacity)
-	..((blocks_air ? temperature : air.temperature))
+	return (blocks_air ? temperature : air.temperature)
```

Comment:
This will break functionality

Reply:
didn't know, i thought they were the same